### DISCUSSION TITLE
Launching on Linux

### DISCUSSION THREAD

**Discussion by pktiuk** [osy/XG_Mobile_Station#11] [Software] [answered]
Has anyone investigated using this dock with Linux? :penguin:      
I guess some alternative to XGMActivator is needed (Or maybe [asusctl](https://gitlab.com/asus-linux/asusctl) plus [supergfxctl](https://gitlab.com/asus-linux/supergfxctl) is enough )?

**Comment by osy** [comment, marked as answer]
Nothing extra is needed. It works with asusctl. Linux doesn't have auto-detect which is what XGMActivator provides
